src/utils/faram.js

```diff
@@ -34,7 +33,6 @@ export const alterResponseErrorToFaramError = (e) => {
     }
 
     const {
-        internalNonFieldErrors = [],
```

Comment:
Why remove this line?

Reply:
was not being used

Reply:
This was used to capture internalNonFieldErrors